version.properties

```diff
@@ -1,2 +1,2 @@
-version=2.0.0-beta.119
+version=2.1.0-beta.119
```

Comment:
Yeah I pondered on the beta number. But since Rafael said how did I miss 2.0.0. I thought that a continuous beta build may help.

I don't have strong opinion on this.